rustport/src/stlab/coarse_priority_queue.rs

```diff
@@ -0,0 +1,120 @@
+use std::cmp::Ordering;
+
+/// A priority queue with three priorities. Elements with equal priorities are popped in FIFO order.
```

Comment:
To me it feels like this description is a little backwards, and it's a FIFO queue with 3 priorities. 🤷‍♂️ I realize they are technically equivalent.